src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -0,0 +1,101 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { AutoSizer } from 'react-virtualized'
+import { Link } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { makeCromwellStatusLine } from 'src/components/job-common'
+import { FlexTable, tableHeight, TooltipCell } from 'src/components/table'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+import { FailuresModal } from 'src/pages/workspaces/workspace/jobHistory/FailuresViewer'
+
+
+const CallTable = ({ namespace, name, submissionId, workflowId, callName, callObjects }) => {
+  const [failuresModalParams, setFailuresModalParams] = useState()
+
+  return div([
+    h(AutoSizer, { disableHeight: true }, [
+      ({ width }) => h(FlexTable, {
+        height: tableHeight({ actualRows: callObjects.length, maxRows: 10.5 }), // The half-row here hints at there being extra rows if scrolled
+        width: _.max([width, 0]),
```

Comment:
Do we need to do math on this? Usually you can just pass the width directly.

Reply:
No - I think this is an unintended carry-over from the old way I was doing this (where I was subtracting from the total page width). Will switch this to just be the width we're passed by the autosizer